roqoqo-test/Cargo.toml

```diff
@@ -19,12 +19,12 @@ crate-type = ["rlib"]
 
 [dependencies]
 qoqo_calculator = { version="1.1" }
-roqoqo = {version="1.2.5", path="../roqoqo", features=["serialize"]}
+roqoqo = {version="1.3.0", path="../roqoqo", features=["serialize"]}
 rand = "0.8"
 nalgebra = "0.32"
 ndarray = { version = "0.15" }
 
 [build-dependencies]
 quote = "1.0"
-syn = { version = "1.0", features = ["full", "visit"] }
+syn = { version = "2.0", features = ["full", "visit"] }
```

Comment:
`roqoqo-derive` is still using `1.0`, is it ok?

Reply:
Good catch fixed

Reply:
In `qoqo-macros` toml as well

Reply:
and `qoqo`